### ISSUE TITLE
Restrict the number of `elif` 

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#70]
Sometimes I see a code like this:

```python
if cond1:
    ...
elif cond2: 
    ...
elif cond3: 
    ...
elif cond4: 
    ...
elif cond5: 
    ...
else:
    ...
```

This is just a `switch` reinvention. Which should not be accepted.
This could also be covered by a `MaxBranches` rule, but I feel like we need to reduce this specific usecase to even less numbers.

What motivation is behind this idea? If you have a complex `switch` it might be a good signal to refactor this piece of code.

So, I guess allowing only two `elif` statements (not counting `else`) is a way to go. You have three? Refactor.

**Comment by sobolevn** [comment]
Related: https://code.joejag.com/2016/anti-if-the-missing-patterns.html
